### DISCUSSION TITLE
Use typings only; for serverless slash command only implementation.

### DISCUSSION THREAD

**Discussion by ChrisBoik** [discordjs/discord.js#5926] [Q&A] [answered]
Is it possible (or recommendable) to import DiscordJS purely for the TS types to handle slash command and other interactions via the interactions URL, in a cloud function for example.
(Essentially without a bot client, simply using the types within webhook style code).

Is there possibly a DiscordJS implementation for this entirely?

**Comment by tipakA** [comment, marked as answer]
There are `discord-api-types`, as a separate package. You Probably should use that instead
